build.gradle

```diff
@@ -14,6 +15,7 @@ buildscript {
 allprojects {
     repositories {
         jcenter()
+        maven { url 'https://maven.google.com' }
```

Comment:
If you use `gradle-4.0-rc-3-all.zip` in the Gradle wrapper, `google()` is available which looks way cleaner. 😄